app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt

```diff
@@ -93,6 +94,33 @@ class HomeMenu(
         }
     }
 
+    val syncedTabsItem = BrowserMenuImageText(
+        context.getString(R.string.synced_tabs),
+        R.drawable.ic_synced_tabs,
+        primaryTextColor
+    ) {
+        onItemTapped.invoke(Item.SyncTabs)
+    }
+
+    private fun getSyncItemTitle(): String {
+        val authenticatedAccount = accountManager.authenticatedAccount
+        val email = accountManager.getAuthAccountEmail()
+
+        return if (authenticatedAccount && email != null) {
+            email
+        } else {
+            context.getString(R.string.sync_menu_sign_in)
+        }
```

Comment:
This is an example of the comment above https://github.com/mozilla-mobile/fenix/pull/19118/files#r617093999

Reply:
We could add `isNullOrEmpty` as part of the email validation.